src/core_modules/capture-core/src/components/QuickSelector/ProgramSelector.component.js

```diff
@@ -1,63 +1,209 @@
 // @flow
+/* eslint-disable */
+
 import React, { Component } from 'react';
 import { withStyles } from 'material-ui-next/styles';
 
 import programs from 'capture-core/metaDataMemoryStores/programCollection/programCollection';
+import OptionSet from 'capture-core/metaData/OptionSet/OptionSet';
```

Comment:
@JoakimSM is the Option and OptionSet general frontend components that should be used in the way below? If so - then you might ask wether they are in the wrong folder: "metadata"?